lib/config/rfc-status-hierarchy.mjs

```diff
@@ -0,0 +1,58 @@
+export const rfcStatusHierarchy = [
+  {
+    name: 'Internet Standard',
+    regex: /internet standard/ig,
+    weight: 7
+  },
+  {
+    name: 'Draft Standard',
+    regex: /draft standard|ds/ig,
+    weight: 6
+  },
+  {
+    name: 'Proposed Standard',
+    regex: /proposed standard|ps/ig,
+    weight: 5
+  },
+  {
+    name: 'Standards Track',
+    regex: /standards track|std/ig,
```

Comment:
this is confused. "std" is a short for "Internet Standard".  The confusion may come from the case we have to deal with for very old RFCs where it's not clear what level it has on the standards track. In those cases, we should do whatever idnits2 currently does.